data_dir/scylla.yaml

```diff
@@ -115,10 +115,12 @@ databases: !mux
         instance_type_db: 'm3.large'
         # Spawn a monitor node
         n_monitor_nodes: 1
+        tcpdump: 0
     scylla:
         db_type: scylla
         # Let's use c3.large since we're using iotune
         # and we'll stress the DB nodes more thoroughly
         instance_type_db: 'c3.large'
         # Spawn a monitor node 
         n_monitor_nodes: 1
+        tcpdump: 0
```

Comment:
This param could be put at the beginning of the config file, also, I think using a boolean would be more fitting, as we are interested in either enabling (true) or disabling it (false).